### PR TITLE
Tickets: Bug fixes and improve form

### PR DESCRIPTION
**Pull request by HoeenCoder**

### CODE REVIEW

File: chat-plugins/tickets.js [hunk 1]
```diff
@@ -230,7 +231,13 @@ exports.pages = {
...
 			buf += `</details><br /><details style="margin: 3px"><summary>I want to appeal a punishment</summary><br /><b>What would you like to appeal?</b><br />`;
-			if (user.locked) {
+			if (isStaff) {
+				// Show all for reference
+				buf += `<details style="margin: 3px"><summary>I want to appeal my permalock</summary>Please make a post in the <a href="http://www.smogon.com/forums/threads/discipline-appeal-rules.3583479/">Discipline Appeal Forums</a> to appeal a permalock.</details>`;
```
Comment (KrisOnline) [inline chat-plugins/tickets.js]: This doesn’t match the format of other smogon links. Smogon links look like `http://www.smogon.com/forums/threads/123456/`

File: chat-plugins/tickets.js [hunk 2]
```diff
@@ -230,7 +231,13 @@ exports.pages = {
...
+				buf += `<details style="margin: 3px"><summary>I want to appeal my lock</summary>If you want to appeal your lock, click the button below and a global staff member will be with you shortly. Alternatively, make a post in <a href="http://www.smogon.com/forums/threads/discipline-appeal-rules.3583479/">Discipline Appeals</a>.<br /><br /><button class="button" name="send" value="/helpticket submit Appeal">Appeal your lock</button></details>`;
+				buf += `<details style="margin: 3px"><summary>I'm locked because I have the same IP as someone I don't recognize</summary>If you are locked under a name you don't recognize, click the button below to call a global staff member so we can check.<br /><br /><button class="button" name="send" value="/helpticket submit IP-Appeal">Appeal IP lock</button></details>`;
+				buf += `<details style="margin: 3px"><summary>I can't talk in chat because of my ISP</summary>Click the button below, and a global staff member will check. <br /><br /><button class="button" name="send" value="/helpticket submit ISP-Appeal">Appeal ISP lock</button></details>`;
+			} else if (user.locked) {
```
Comment (Zarel) [inline chat-plugins/tickets.js]: `|| isStaff`.

Better than code duplication.

**Comment by Zarel** [comment]
I suppose I'll merge and fix this myself...

File: chat-plugins/tickets.js (hunk 1 above)
Comment (panpawn) [reply chat-plugins/tickets.js]: That's probably bad, they should probably just be ``https://smogon.com/...``

**Pull request merged**